From 9b89c644e3588dd48d24ff9294d0365d79315a24 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 29 Jan 2024 17:25:47 +0200
Subject: [PATCH] [SW-170975] scripts: update security tgz

Since NIC fits are needed to be secured and signed. Add them to security
tgz.

Signed-off-by: Guy Nisan <gnisan@habana.ai>
Change-Id: I782683857443f63866230abc8e413e24a987971c
---

diff --git a/scripts/make/gaudi2/Makefile b/scripts/make/gaudi2/Makefile
index e33ea3d..5ae071a 100644
--- a/scripts/make/gaudi2/Makefile
+++ b/scripts/make/gaudi2/Makefile
@@ -115,7 +115,10 @@
 	${HL_DEV}-psu-ctrl.itb			\
 	${HL_DEV}-rma_erase.itb			\
 	${HL_DEV}-rma_status.itb		\
-	${HL_DEV}-efuse_read.itb
+	${HL_DEV}-efuse_read.itb		\
+	${HL_DEV}-nic-default.itb		\
+	${HL_DEV}-nic-no_ext.itb
+
 
 .PHONY: gaudi2-security-fit-create
 gaudi2-security-fit-create: gaudi2-eeprom-read-create gaudi2-eeprom-create-225c		\
@@ -123,7 +126,8 @@
 				gaudi2-update-svn-create gaudi2-fw-upd-closure-create	\
 				gaudi2-sram-sf-access-create gaudi2-psu-ctrl-create	\
 				gaudi2-rma-erase-create gaudi2-rma-status-create	\
-				gaudi2-efuse-read-create gaudi2-fw-upd-open-create
+				gaudi2-efuse-read-create gaudi2-fw-upd-open-create	\
+				gaudi2-nic-en-create
 
 	@$(call hl-check-files,$(GAUDI2_SECURITY_FITS))
 	@tar czf $(IMAGES_DIR)/${HL_DEV}-sec-fit.tgz -C $(IMAGES_DIR)/ $(GAUDI2_SECURITY_FITS)
@@ -133,8 +137,6 @@
 	sshpass -p$(HL_PASSWD) $(SCP) $(IMAGES_DIR)/${HL_DEV}-sec-fit.tgz root@$(HL_INSTALL_LAB_IP):/lib/firmware/habanalabs/$(HL_DEV)/
 
 GAUDI2_INTERNAL_FITS := ${HL_DEV}-nic-en_hls2.itb	\
-	${HL_DEV}-nic-en.itb		\
-	${HL_DEV}-nic-dis.itb		\
 	${HL_DEV}-eeprom.itb		\
 	${HL_DEV}-efuse_write.itb	\
 	${HL_DEV}-write_tpm_devid.itb		\
@@ -151,7 +153,7 @@
 	${HL_DEV}-secureboot_config_no_rev.itb
 
 .PHONY: gaudi2-internal-fit-create
-gaudi2-internal-fit-create: gaudi2-nic-en-create gaudi2-efuse-write-create-default	\
+gaudi2-internal-fit-create: gaudi2-efuse-write-create-default	\
 				gaudi2-eeprom-create-default gaudi2-write-tpm-devid-create	\
 				gaudi2-slt-set-create gaudi2-security-enable-create	\
 				gaudi2-otp-create gaudi2-otp-local-create	\
diff --git a/scripts/make/gaudi3/Makefile b/scripts/make/gaudi3/Makefile
index 1a9c1bc..c91d64d 100644
--- a/scripts/make/gaudi3/Makefile
+++ b/scripts/make/gaudi3/Makefile
@@ -59,9 +59,7 @@
 gaudi3-efuse-write-create-default: gaudi3-efuse-write-set-default gaudi3-efuse-write-create
 	@echo "default efuse write created"
 
-GAUDI3_INTERNAL_FITS := ${DEV_PREFIX}-nic-en.itb	\
-	${DEV_PREFIX}-nic-dis.itb			\
-	${DEV_PREFIX}-eeprom.itb			\
+GAUDI3_INTERNAL_FITS :=	${DEV_PREFIX}-eeprom.itb	\
 	${DEV_PREFIX}-eeprom-recovery.itb		\
 	${DEV_PREFIX}-security_enable.itb		\
 	${DEV_PREFIX}-secureboot_config.itb		\
@@ -78,7 +76,7 @@
 
 
 .PHONY: gaudi3-internal-fit-create
-gaudi3-internal-fit-create: gaudi3-nic-en-create gaudi3-eeprom-create-default	\
+gaudi3-internal-fit-create: gaudi3-eeprom-create-default	\
 				gaudi3-eeprom-create-recovery			\
 				gaudi3-security-enable-create			\
 				gaudi3-secureboot-config-create			\
@@ -109,7 +107,9 @@
 	${DEV_PREFIX}-sram_sf-close.itb			\
 	${DEV_PREFIX}-rma_erase.itb			\
 	${DEV_PREFIX}-rma_status.itb			\
-	${DEV_PREFIX}-efuse_read.itb
+	${DEV_PREFIX}-efuse_read.itb			\
+	${DEV_PREFIX}-nic-default.itb
+
 
 .PHONY: gaudi3-security-fit-create
 gaudi3-security-fit-create: gaudi3-eeprom-read-create					\
@@ -117,7 +117,7 @@
 				gaudi3-update-svn-create gaudi3-fw-upd-closure-create	\
 				gaudi3-sram-sf-access-create				\
 				gaudi3-rma-erase-create gaudi3-rma-status-create	\
-				gaudi3-efuse-read-create
+				gaudi3-efuse-read-create gaudi3-nic-en-create
 
 	@$(call hl-check-files,$(GAUDI3_SECURITY_FITS))
 	@tar czf $(IMAGES_DIR)/${DEV_PREFIX}-sec-fit.tgz -C $(IMAGES_DIR)/ $(GAUDI3_SECURITY_FITS)
